buildkite/src/Jobs/Promote/AutoPromoteNightly.dhall

```diff
@@ -47,9 +47,7 @@ let targetVersion =
       ->  \(commit : Text)
       ->  \(latestGitTag : Text)
       ->  \(todayDate : Text)
-      ->  "${latestGitTag}-${todayDate}-${DebianVersions.lowerName
-                                            codename}-${DebianChannel.lowerName
-                                                          channel}"
+      ->  "${latestGitTag}-${todayDate}"
```

Comment:
We don't need suffix for master/compatible/develop since we are pushing debian to channel with the same name